### ISSUE TITLE
libjpl.dylib library is missing in SWI-Prolog 8.2.3 for macOS

### ISSUE THREAD

**Issue by jnidzwetzki** [SWI-Prolog/packages-jpl#91]
Hello,

I have noticed that the `libjpl.dylib` library is missing in the current build of swi-prolog 8.2.3 for [macOS](https://www.swi-prolog.org/download/stable/bin/swipl-8.2.3-1.x86_64.dmg); the swi-prolog build 8.2.2 contains the library. I could not find any information about this in the changelog. Is the library missing for a reason, or is this a bug in the macOS build?

```bash
###
# Downloaded and mounted the last versions of swi-prolog (8.0.3, 8.2.0, 8.2.1, 8.2.2, 8.2.3)
###
$ ls -ld /Volumes/swipl-*
drwxr-xr-x  6 kristofnidzwetzki  staff  272 19 Jun  2019 /Volumes/swipl-8.0.3-1.x86_64
drwxr-xr-x  7 kristofnidzwetzki  staff  306 27 Mai  2020 /Volumes/swipl-8.2.0-1.x86_64
drwxr-xr-x  7 kristofnidzwetzki  staff  306 29 Jun  2020 /Volumes/swipl-8.2.1-1.x86_64
drwxr-xr-x  7 kristofnidzwetzki  staff  306 27 Okt 13:11 /Volumes/swipl-8.2.2-1.x86_64
drwxr-xr-x  7 kristofnidzwetzki  staff  306 23 Nov 09:31 /Volumes/swipl-8.2.3-1.x86_64

###
# All versions contain the libjpl.dylib library, except swi-prolog 8.2.3
###
$ find /Volumes/swipl-8.* | grep libjpl.dylib                                      
/Volumes/swipl-8.0.3-1.x86_64/SWI-Prolog.app/Contents/swipl/lib/x86_64-darwin/libjpl.dylib
/Volumes/swipl-8.2.0-1.x86_64/SWI-Prolog.app/Contents/swipl/lib/x86_64-darwin/libjpl.dylib
/Volumes/swipl-8.2.1-1.x86_64/SWI-Prolog.app/Contents/swipl/lib/x86_64-darwin/libjpl.dylib
/Volumes/swipl-8.2.2-1.x86_64/SWI-Prolog.app/Contents/swipl/lib/x86_64-darwin/libjpl.dylib
```

Thank you very much in advance.

Best Regards
   Jan

**Comment by JanWielemaker** [comment]
Yip.  This is a build issue for the MacOS binary due to a MacOS change in the program that returns the current Java version.  It has been resolved in the devel series and the git source for the stable series.  You can probably use the 8.2.2 version, but this might need some tweaking with otools.   Will be fixed with 8.2.4, which probably isn't too far away.

**Comment by jnidzwetzki** [comment]
@JanWielemaker Thank you very much for the quick response.
